### ISSUE TITLE
Vulnerabilities in Go dependencies

### ISSUE THREAD

**Issue by bemeyert** [helm/helm#9956]
Output of `helm version`: 3.6.3

We run a scan with `trivy` on our images containing above Helm version and it showed us that there are dependencies with vulnerabilities of the severity high:
```
usr/local/bin/helm (gobinary)
=============================
Total: 3 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 3, CRITICAL: 0)

+--------------------------------+------------------+----------+------------------------------------+------------------------------------------+---------------------------------------+
|            LIBRARY             | VULNERABILITY ID | SEVERITY |         INSTALLED VERSION          |              FIXED VERSION               |                 TITLE                 |
+--------------------------------+------------------+----------+------------------------------------+------------------------------------------+---------------------------------------+
| github.com/docker/distribution | CVE-2017-11468   | HIGH     | v0.0.0-20191216044856-a8371794149d | v2.7.0-rc.0+incompatible                 | docker-distribution: Does not         |
|                                |                  |          |                                    |                                          | properly restrict the amount          |
|                                |                  |          |                                    |                                          | of content accepted from a...         |
|                                |                  |          |                                    |                                          | -->avd.aquasec.com/nvd/cve-2017-11468 |
+--------------------------------+------------------+          +------------------------------------+------------------------------------------+---------------------------------------+
| github.com/opencontainers/runc | CVE-2019-16884   |          | v0.1.1                             | v1.0.0-rc8.0.20190930145003-cad42f6e0932 | runc: AppArmor/SELinux bypass         |
|                                |                  |          |                                    |                                          | with malicious image that             |
|                                |                  |          |                                    |                                          | specifies a volume at /proc...        |
|                                |                  |          |                                    |                                          | -->avd.aquasec.com/nvd/cve-2019-16884 |
+                                +------------------+          +                                    +------------------------------------------+---------------------------------------+
|                                | CVE-2019-19921   |          |                                    | v1.0.0-rc9.0.20200122160610-2fc03cc11c77 | runc: volume mount race condition     |
|                                |                  |          |                                    |                                          | with shared mounts leads to           |
|                                |                  |          |                                    |                                          | information leak/integrity...         |
|                                |                  |          |                                    |                                          | -->avd.aquasec.com/nvd/cve-2019-19921 |
+--------------------------------+------------------+----------+------------------------------------+------------------------------------------+---------------------------------------+
```
Could you please update above dependencies? Many thanks.

**Comment by ryansann** [comment]
We're also seeing these CVEs in our image scans that use helm v3.6.3

**Comment by bacongobbler** [comment]
Hi @bemeyert and @ryansann. In the future, please reach out using the proper channels to disclose potential security risks. Please read [CONTRIBUTING.md](https://github.com/helm/helm/blob/main/CONTRIBUTING.md#reporting-a-security-issue) and/or [SECURITY.md](https://github.com/helm/helm/blob/main/SECURITY.md) for more information.

Okay, back to the issue.

Looking at Helm's source code, Helm does not use runc or distribution directly. They are sub-dependencies of [the oras project](https://github.com/oras-project/oras) which is used to push and pull artifacts to and from an OCI registry.

ORAS utilizes containerd's libraries (and by extension, runc/distribution) to fetch and push OCI manifests to OCI registries.

Helm does not use runc to execute containers at any point during its execution. Additionally, Helm does not serve a container registry endpoint for hosting artifacts.

In other words, the CVEs described above do not apply to Helm.

Please feel free to submit a PR to bump Helm's either ORAS or the sub-dependencies to a higher version. As it currently stands though there does not appear to be any security risk with the current version.

Again, I would like to re-iterate to please use the proper security channels next time. It's important to us to ensure that potential security risks are not publicly disclosed until we have a patch available for general consumption.

Thanks.

**Comment by bacongobbler** [comment]
It is also worth noting this was reported earlier in https://github.com/helm/helm/issues/7846. You can follow the upstream ticket here: https://github.com/oras-project/oras/issues/156

Closing as a duplicate of #7846, which has the same resolution: bump oras's depenendencies. Once that is done, we can update ORAS to the latest version to receive the update. But these do not pose any security risk to either project at this time.

**Comment by bemeyert** [comment]
@bacongobbler Mea culpa! You got me here. I'm quite new in the K8s business and my "sensitivity" regarding security is slowly rising. So, next time I'll do better. Many thanks for your help.
